### PR TITLE
fix:camperchan user setting description to be more descriptive asking for URL and HEX value

### PR DESCRIPTION
**Pull request by RGHANILOO**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/commands/userSettings.ts [hunk 1]
```diff
@@ -13,24 +13,30 @@ export const userSettings: Command = {
...
+        .setDescription(
+          "The avatar to appear on your profile card must be a URL that points to an image."
+        )
     )
     .addStringOption((option) =>
       option
         .setName("background-colour")
         .setDescription(
-          "The semi-transparent background colour for your profile card."
+          "The semi-transparent background color for your profile card must be a hex value."
```
Comment (naomi-lgbt) [inline src/commands/userSettings.ts]:
```suggestion
          "The semi-transparent background color for your profile card must be a 6-digit hex value."
```

File: src/commands/userSettings.ts [hunk 2]
```diff
@@ -13,24 +13,30 @@ export const userSettings: Command = {
...
+        .setDescription(
+          "The background image for your profile card must be a URL that points to an image."
+        )
     )
     .addStringOption((option) =>
       option
         .setName("colour")
-        .setDescription("The colour for the text on your profile card.")
+        .setDescription(
+          "The color for the text on your profile card must be a hex value."
```
Comment (naomi-lgbt) [inline src/commands/userSettings.ts]:
```suggestion
          "The color for the text on your profile card must be a 6-digit hex value."
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
The validator doesn't allow for 3 character hex codes, so we might as well clarify that too :3

File: src/commands/userSettings.ts (hunk 1 above)
Comment (RGHANILOO) [reply src/commands/userSettings.ts]: commit  #9f929a0 fixes this

File: src/commands/userSettings.ts (hunk 2 above)
Comment (RGHANILOO) [reply src/commands/userSettings.ts]: commit  #9f929a0 fixes this

**Comment by RGHANILOO** [comment]
> The validator doesn't allow for 3 character hex codes, so we might as well clarify that too :3

that makes sense, more explicit . this is all done now. 😄

**Review by naomi-lgbt — APPROVED** [review]
Beautiful! Let's get this merged and deployed. 💜

**Pull request merged**